conseil-api/src/test/scala/tech/cryptonomic/conseil/api/routes/platform/data/bitcoin/BitcoinDataQueriesTest.scala

```diff
@@ -104,6 +104,31 @@ class BitcoinDataQueriesTest
           result.value.size shouldBe 3
         }
       }
+
+      "return proper number of accounts, while fetching all of accounts" in {
+        // given
+        dbHandler.run(Tables.Blocks ++= blocks).isReadyWithin(5.seconds) shouldBe true
+        dbHandler.run(Tables.Transactions ++= transactions).isReadyWithin(5.seconds) shouldBe true
+        dbHandler.run(Tables.Inputs ++= inputs).isReadyWithin(5.seconds) shouldBe true
+        dbHandler.run(Tables.Outputs ++= outputs).isReadyWithin(5.seconds) shouldBe true
+
+        whenReady(sut.fetchAccounts(Query.empty)) { result =>
+          result.value.size shouldBe 3
+        }
+      }
+
+      "return proper account by address" in {
+        // given
+        dbHandler.run(Tables.Blocks ++= blocks).isReadyWithin(5.seconds) shouldBe true
+        dbHandler.run(Tables.Transactions ++= transactions).isReadyWithin(5.seconds) shouldBe true
+        dbHandler.run(Tables.Inputs ++= inputs).isReadyWithin(5.seconds) shouldBe true
+        dbHandler.run(Tables.Outputs ++= outputs).isReadyWithin(5.seconds) shouldBe true
+
+        whenReady(sut.fetchAccountByAddress("script_pub_key_address_2")) { result =>
+          result.value should (contain key "address" and contain value output2.scriptPubKeyAddresses)
+          //TODO Once we will replace BigDecimal into something better, we should compare 'value' field here as well
```

Comment:
Please expand on this note: give a reference to what is the BigDecimal in this case, what is the impact and limitations that this entails on the current tests, and how it's supposed to be fixed in the future.

Please write comments like someone other than you would need to take charge of this task, so he/she would need all the details to manage by him/herself